.circleci/scripts/upload_cloud_watch_to_s3.py

```diff
@@ -27,16 +27,14 @@ def upload_cloud_watch_logs(log_group_name: str, bucket_name: str, prefix: str,
         except botocore.exceptions.ClientError as error:
             if error.response['Error']['Code'] == 'LimitExceededException':
                 print(f"Got an error possibly due to concurrent uploads: {error}.")
+                attempts += 1
```

Comment:
I dont think we need to move the code around. At least the way it was we could add
```
elif error.response['Error']['Code'] == 'OtherAnnoyingFlake':
```
too. I think this is more about us not waiting long enough.